services/web/server/src/simcore_service_webserver/wallets/_handlers.py

```diff
@@ -83,6 +84,7 @@ class WalletsPathParams(StrictRequestParams):
 
 
 @routes.post(f"/{VTAG}/wallets", name="create_wallet")
+@requires_dev_feature_enabled  # NOTE: one wallet per user+product. SEE _events.py:_auto_add_default_wallet
```

Comment:
Q: I am thinking whether `requires_dev_feature_enabled` makes sense in this use case. I have a feeling that this feature might be required in the staging soon after it is tested.

Reply:
Aha yes, you are right! thanks for explaining.